Dockerfile

```diff
@@ -1,5 +1,14 @@
 # Copyright 2021 VMware, Inc.
 # SPDX-License-Identifier: BSD-2-Clause
+FROM golang:1.16 as builder
+ARG VERSION
```

Comment:
This will disappear from the diff when I can rebase after the relok8s Dockerfile is merged